web-frontend/modules/database/components/docs/APIDocsExample.vue

```diff
@@ -25,7 +25,7 @@
               type.toLowerCase() === 'delete',
           }"
         >
-          {{ type | uppercase }}
+          {{ type.toUpperCase() }}
```

Comment:
For your information, the filters had been ported to `$filters.uppercase()`. You don't neet to change anything if it works but it's FYI.